### ISSUE TITLE
the reslut of MNN and onnx is different from pytorch

### ISSUE THREAD

**Issue by cendelian** [clovaai/rexnet#14]
HI
Thansk for for your great work.
I use rexnet to train classify net.
and I transfer it from Pytorch -> ONNX -> MNN.
the result of onnx and mnn are the same, but they are different from pytorch result.

**Does rexnet can transform to ONNX normally and maintain result same?**

**Comment by dyhan0920** [comment]
@cendelian, I am sure that if the problem has resolved. Sorry for the late reply. 

I did not verify converting ONNX models to MNN works well or not, but if the accuracy after converting looks abnormal, the conversion may not be successfully done. I also converted the current models to ONNX without any problems due to my personal project,  However, I found transforming ONNX models to other frameworks (I used Keras, TF2) was not straightforward.

I found slight changes are needed to convert ONNX models to other frameworks successfully, and I tested the converted models when doing finetuning. I will upload the modified models late today.
